lib/plugins/standalone.js

```diff
@@ -73,7 +73,7 @@ module.exports = class Standalone {
     }
     await fse.remove(BINARY_TMP_PATH);
     await pipeline(standaloneResponse.body, fs.createWriteStream(BINARY_TMP_PATH));
-    await fse.rename(BINARY_TMP_PATH, BINARY_PATH);
+    await fs.promises.rename(BINARY_TMP_PATH, BINARY_PATH);
```

Comment:
Is there a reason as to why the same change was not applied here? (`fs.promises` -> `fsp`) ?

Reply:
@pgrzesik In this file, `fs` was imported earlier and used as `fs.createWriteStream` (L75). I don't think we can use `fs.promises` with `createWriteStream`, so couldn't import `fs` as `fsp` for this case. Let me know if you have any other thoughts.

Reply:
We can still import both `const fs = require('fs')` and in next line `const fsp = require('fs').promises`